mne/viz/_3d.py

```diff
@@ -212,8 +218,7 @@ def _plot_mri_contours(mri_fname, surf_fnames, orientation='coronal',
     trans[:3, -1] = [n_sag // 2, n_axi // 2, n_cor // 2]
 
     for surf_fname in surf_fnames:
-        surf = dict()
-        surf['rr'], surf['tris'] = read_surface(surf_fname)
+        surf = read_surface(surf_fname, return_dict=True)[-1]
         # move back surface to MRI coordinate system
         surf['rr'] = nib.affines.apply_affine(trans, surf['rr'])
```

Comment:
It should have been fixed by my commit yesterday, it came from `complete_surface_info` which now has `verbose=False`